app/src/main/res/layout/activity_qr_code_connect.xml

```diff
@@ -33,7 +33,7 @@
             android:layout_marginTop="8dp"
             android:layout_marginEnd="8dp"
             android:layout_marginBottom="8dp"
-            android:text="@string/scanQRCode"
```

Comment:
Should we remove this string? I think its not used anymore.

Reply:
yes :)